src/lib/utils/error-logger.ts

```
@@ -0,0 +1,71 @@
+   * Log info message with context
+   */
+  static async logInfo(context: string, message: string, processType: string = 'SYSTEM'): Promise<void> {
+    try {
+      await invoke('log_translation_process', {
+        message: `[${context}] ${message}`
+      });
+    } catch (error) {
+      console.error('[ErrorLogger] Failed to send info to backend:', error);
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)